src/main/resources/locales/cy/translation/type-of-claim.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Type of claim",
```

Comment:
Should this be 'Welsh Translation required'?

Reply:
Fixed all of them, found more:)